storage/grpc_client.go

```diff
@@ -95,6 +95,7 @@ func defaultGRPCOptions() []option.ClientOption {
 			option.WithEndpoint(host),
 			option.WithGRPCDialOption(grpc.WithInsecure()),
 			option.WithoutAuthentication(),
+			WithDisabledClientMetrics(),
```

Comment:
Do we need to do the same for HTTP?

Reply:
Not yet, metrics are only available in gRPC.